applications/StructuralMechanicsApplication/tests/test_StructuralMechanicsApplication.py

```diff
@@ -306,6 +308,8 @@ def AssembleTestSuites():
     # Eigenvalues Postprocessing Process test
     smallSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([TTestPostprocessEigenvaluesProcess]))
     smallSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([TestEigenSolverWithConstraints]))
+    # Custom Scipy Solver test
+    nightSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([TestCustomScipyBaseSolver]))
```

Comment:
how long does this test take on your machine?

Reply:
I believe to remember it was relatively long, although it is pretty much the same example as in the normal EigensolverTest.  I let you know, but first have to recompile.

Reply:
It takes 0.5sec

Reply:
ok then nightly is appropriate